src/osd/OpRequest.h

```diff
@@ -180,7 +181,7 @@ struct OpRequest : public TrackedOp {
 
 private:
   void mark_flag_point(uint8_t flag, const char *s);
-  void mark_flag_point_string(uint8_t flag, const std::string& s);
+  void mark_flag_point_string(uint8_t flag, const char* s);
```

Comment:
You can't actually reuse this.  The only caller passes in a genuinely heap allocated string meant to be passed to tracepoint().  That usage is safe, but storing a pointer to it would not be.

Reply:
I got it